tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/conversion/MavenConversionTest.java

```diff
@@ -270,5 +252,17 @@ private void finishConversionDialog(){
 			new WaitWhile(new JobIsRunning(),TimePeriod.VERY_LONG);
 		}
 	}
-
+	
+	private List<String> getJavaBuildPathLibraries(PropertyDialog pd) {
+		pd.select("Java Build Path");
```

Comment:
Instead of this use something similar with select, see https://github.com/eclipse/reddeer/blob/5791383f991e4b2503988d5db59cc85b93055cfc/tests/org.eclipse.reddeer.eclipse.test/src/org/eclipse/reddeer/eclipse/test/jdt/ui/preferences/BuildPathsPropertyPageTest.java#L81

(to hide "Java Build Path" string in reddeer)

Reply:
Changed.